supergloo/docs/mesh/tutorials-4-security.md

```diff
@@ -0,0 +1,287 @@
+microservice communication with the other 3 services. 
+
+Let's start by enabling access restrictions with a single SecurityRule. The rule we'll create will enable SuperGloo's policy features without whitelisting any communication.
+
+Run the following command to create the SecurityRule in *interactive mode*:
+
+```bash
+# run supergloo cli in interactive mode
+supergloo apply securityrule -i
+```
+```
+? name for the Security Rule:  enable-security
+? namespace for the Security Rule:  default
+? create a source selector for this rule?  [y/N]:  y
+? what kind of selector would you like to create?  Upstream Selector
+? add an upstream (choose <done> to finish):  supergloo-system.default-productpage-9080
+? add an upstream (choose <done> to finish):  <done>
+? create a destination selector for this rule?  [y/N]:  Y
+? what kind of selector would you like to create?  Upstream Selector
+? add an upstream (choose <done> to finish):  supergloo-system.default-productpage-9080
+? add an upstream (choose <done> to finish):  <done>
+? select a target mesh to which to apply this rule supergloo-system.istio
+? enter a comma-separated list of HTTP methods to allow for this rule, e.g.: GET,POST,PATCH (leave empty to allow all):
+? enter a comma-separated list of HTTP paths to allow for this rule, e.g.: /api,/admin,/auth (leave empty to allow all):
+```
+
+* The equivalent noninteractive version of this command would be
+
+```bash
+supergloo apply securityrule \
+    --name enable-security \
+    --namespace default \
+    --source-upstreams supergloo-system.default-productpage-9080 \
+    --dest-upstreams supergloo-system.default-productpage-9080 \
+    --target-mesh supergloo-system.istio
+```
+
+* Or, using `kubectl`:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: supergloo.solo.io/v1
+kind: SecurityRule
+metadata:
+  name: enable-security
+  namespace: default
+spec:
+  destinationSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  sourceSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  targetMesh:
+    name: istio
+    namespace: supergloo-system
+EOF
+``` 
+
+Now that we've created our first SecurityRule, Access-Control should be enforced in our cluster.
+Now try refreshing the Product Page. 
+
+> Note: it may take up to a minute for configuration to propagate to all the Istio sidecars.
+
+We should see that the details and reviews sections of the UI have turned into error messages:
+
+![Bookinfo Product Page](../../img/bookinfo-all-error.png "Bookinfo Product Page")
+
+This confirms that the `productpage` service can no longer send requests to `details` or `reviews`.
+
+Let's add a rule to permit the `productpage` to communicate with `details`.
+
+* Using `supergloo` CLI:
+
+```bash
+supergloo apply securityrule \
+    --name productpage-to-details \
```

Comment:
So this rule didn't see to do anything. Is the "name" of the SecurityRule supposed to match the previous one?

Reply:
it may take a few minutes for istio rbac to roll out to the proxies. the name is correct